composer.json

```diff
@@ -64,7 +63,6 @@
         "friendsofsymfony/rest-bundle": "^2.1 || ^3.0",
         "jms/serializer-bundle": "^2.0 || ^3.0",
         "matthiasnoback/symfony-dependency-injection-test": "^4.1.1",
-        "nelmio/api-doc-bundle": "^2.4",
```

Comment:
you can remove fos rest , jms serializer and nelmio too. Keep in mind the suggest section too.

Reply:
done